drivers/note/noteram_driver.c

```diff
@@ -105,10 +108,24 @@ static struct noteram_info_s g_noteram_info =
 static struct noteram_taskname_s g_noteram_taskname;
 #endif
 
+static struct note_driver_ops_s noteram_ops =
+{
+  .write = sched_noteram_write
```

Comment:
can we avoid designated initializer?

Reply:
We may need to record some information when the system was first started, when the drivers has not been initialized

So we need to specify a default note driver by going to global variables

Reply:
I may have misunderstood, the structure has been initialized in C89 style